src/coefficients/coefficient_extensions.hpp

```diff
@@ -243,6 +243,6 @@ class TransformedScalarCoefficient : public mfem::Coefficient {
   std::function<double(const double, const double)> bi_function_;
```

Comment:
This isn't really an MFEM extension, I.e. it is not derived from an MFEM class. Do we still want to label it as an MFEM extension?

Reply:
Thoughts @white238 ?

Reply:
Just to log the output of our meeting on this topic the `serac::mfem_extensions` namespace will cover two cases:

1) Extended MFEM classes
2) Functionality we hope will go into MFEM

The idea is to keep their case style so it is less of a rework when they get moved into MFEM and also a class with mixed casing is very awkward.